features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/ApiConsole/TryOutController.jsx

```diff
@@ -332,6 +332,17 @@ function TryOutController(props) {
             case 'password':
                 setPassword(value);
                 break;
+            case 'accessToken':
+                if (securitySchemeType === 'API-KEY' && selectedKeyType === 'PRODUCTION') {
```

Comment:
If these are constants and used in some other paces, Better to move them to a constant variable and refer to the variable when need the value